setup.py

```diff
@@ -81,7 +81,7 @@ def get_git_sha() -> str:
         "croniter>=0.3.28",
         "cron-descriptor",
         # snowflake-connector-python as of 3.7.0 doesn't support >=42.* therefore lowering the min to 41.0.2
-        "cryptography>=41.0.2, <43.0.0",
+        "cryptography>=42.0.5, <43.0.0",
```

Comment:
We need to keep this at `41.0.2` because there are extra dependencies that colide with it

Reply:
you mean to roll back the changes and only add the 42.0.5 version in base.in instead for now - right?